src/EnergyPlus/HVACUnitarySystem.cc

```diff
@@ -2446,7 +2446,7 @@ namespace HVACUnitarySystem {
 			PrintFlag = true;
 			TempSize = UnitarySystem( UnitarySysNum ).DesignMaxOutletTemp;
 			FieldNum = 18; // N18, \field Maximum Outdoor Dry-Bulb Temperature for Supplemental Heater Operation
-			SizingString = UnitarySystemNumericFields( UnitarySysNum ).FieldNames( FieldNum ) + " [m3/s]";
+			SizingString = UnitarySystemNumericFields( UnitarySysNum ).FieldNames( FieldNum ) + " [C]";
```

Comment:
@EnergyArchmage I agree with the units change, but it looks like this should be field 17, not 18.  From the IDD
```
  N17, \field Maximum Supply Air Temperature
       \type real
       \units C
       \autosizable
       \default 80.0
       \note Enter the maximum supply air temperature leaving the heating coil.
  N18, \field Maximum Outdoor Dry-Bulb Temperature for Supplemental Heater Operation
       \type real
       \units C
       \default 21.0
       \note Enter the maximum outdoor dry-bulb temperature for supplemental heater operation.
```